### ISSUE TITLE
Orders admin interface workflow

### ISSUE THREAD

**Issue by Gouthamve** [reactioncommerce/reaction#361]
I think the current order handling on the admin side is incomplete.

1. I cannot generate an invoice until I add tracking
2. The downloaded invoice is lost (It cannot be retrieved again).  
2.b Also the invoice generated could be better. (minor #307)
3. There is complete packing after the tracking is added which does not make sense.

The tracking should come after we ship. So the invoice and shipping label should be done first and the tracking later. Also there should be an option to download the invoice whenever needed.

And keeping in mind #285 #290 I think each order should have its own page where all this is handled. Instead of handling everything on the orders admin interface.

**Comment by aaronjudd** [comment]
Agree -> the "kanban" order handling layout was a concept.. but I don't think it's really going to work -> I'm thinking more of a list, (maybe a bit more like the shipping stuff) each order can have a few "quick helpers" but being able to just go to a replica of the order, with all the necessary steps and also a print layout.  Also think we need to consider #278.  My original thinking on the workflow is that you should be able to add, remove, reorder both cart and order workflows to accommodate different processes (such as submission to a fulfillment center) - UI will need to reflect those changes.

**Comment by aaronjudd** [comment]
Closing in favor of new tickets related to specific order workflow improvements, now that we've refactored order workflows.
